README.md

```diff
@@ -67,24 +90,45 @@ for dataset in client.list_remote_datasets():
     print(dataset.slug, dataset.image_count)
 ```
 
-#### Upload data to a [remote] project (images/videos)
 
-Uploads data to an existing remote project.
-It takes the dataset slug and a list of file names of images/videos to upload as parameters.
+#### Create/remove a dataset 
+
+Dataset creation is handled by the client:
 
 ```python
 from darwin.client import Client
 
 client = Client.default()
-dataset = client.get_remote_dataset(slug="example-dataset")
-progress = dataset.upload_files(["test.png", "test.mp4"])
-for _ in progress():
-    print("file uploaded")
+client.create_dataset(name="This Is My New Dataset")
 ```
 
-#### Pull a [remote] project
+Whereas dataset removal is handled directly by the dataset themselves:
+
+```python
+from darwin.client import Client
 
-Downloads a remote project, images and annotations, in the projects directory (specified in the authentication process [default: ~/.darwin/projects]).
+client = Client.default()
+dataset = client.get_remote_dataset(slug="this-is-my-new-dataset")
+dataset.remove()
+```
+
+
+#### Upload/download data to/from a remote dataset
+
+To upload data to an existing remote project there are several solutions.
+A simple one is to update the local folder where the dataset is located and then upload the
+files to the remote dataset. 
+
+```python
+from darwin.client import Client
+
+client = Client.default()
+dataset = client.get_remote_dataset(slug="example-dataset")
+progress = dataset.push()
+```
+
```

Comment:
The push function raises an error (it seems to assume by default that we're uploading videos?):
```





  0%|          | 0/1 [00:00<?, ?it/s]

---------------------------------------------------------------------------
InsufficientStorage                       Traceback (most recent call last)
<ipython-input-131-0559e2aa2f49> in <module>
----> 1 rd.push()

/data/blanca/darwin-py/darwin/dataset/remote_dataset.py in push(self, blocking, multi_threaded, extensions_to_exclude, fps, files_to_upload)
    110         if blocking:
    111             exhaust_generator(
--> 112                 progress=progress, count=count, multi_threaded=multi_threaded
    113             )
    114             return None, count

/data/blanca/darwin-py/darwin/dataset/utils.py in exhaust_generator(progress, count, multi_threaded)
    337 
    338         with mp.Pool(mp.cpu_count()) as pool:
--> 339             for f in progress:
    340                 pool.apply_async(_f, args=(f,), callback=update)
    341             pool.close()

/data/blanca/darwin-py/darwin/dataset/upload_manager.py in add_files_to_dataset(client, dataset_id, filenames, fps)
     56                 "image_filenames": [Path(image).name for image in images],
     57                 "videos": [
---> 58                     {"fps": fps, "original_filename": Path(video).name} for video in videos
     59                 ],
     60             },

/data/blanca/darwin-py/darwin/client.py in put(self, endpoint, payload, retry)
    113             error_code = response.json()["errors"]["code"]
    114             if error_code == "INSUFFICIENT_REMAINING_STORAGE":
--> 115                 raise InsufficientStorage()
    116 
    117         if response.status_code != 200:

InsufficientStorage: 
```

Reply:
I had this error already. Apparently it is because of the team permissions limit. I solved it by creating a new team and pushing there (in production). The real solution would be to pay for a higher level storage capability (or so I've been told)